src/components/Home/HomeDetail.tsx

```diff
@@ -19,7 +22,10 @@ import {
   PageTitle,
   PageSubTitle,
 } from '../common'
-import { getMakerDaoData, getPollsData } from '../../utils/makerdao'
+import { getPollsData, getMKRSupply } from '../../utils/makerdao'
+import { getPollData } from '../../components/PollDetails/data'
+import { getPollsBalances } from '../List/helpers'
```

Comment:
I would move this functions to a more generic place, since we are using them in several places